activities/ingest_queue_populate.py

```diff
@@ -98,24 +112,39 @@ def clear_queue(arn):
     client.purge_queue(QueueUrl = arn)
     time.sleep(60)
 
-def split_args(args):
-    range_ = lambda v: range(args[v + '_start'], args[v + '_stop'], args[v + '_tile_size'])
 
-    for t in range_('t'):
-        for z in range_('z'):
-            args_ = args.copy()
+def get_tile_count(args):
+    tile_size = lambda v: args[v + "_tile_size"]
+    extent = lambda v: args[v + '_stop'] - args[v + '_start']
+    num_tiles_in = lambda v: math.ceil(extent(v) / tile_size(v))
+
+    x_extent = args['x_stop'] - args['x_start']
```

Comment:
Why not use the `extent` lambda you define above?

Reply:
Updated.